spark/src/main/java/org/opensearch/sql/spark/dispatcher/BatchQueryHandler.java

```diff
@@ -69,8 +68,6 @@ public String cancelJob(AsyncQueryJobMetadata asyncQueryJobMetadata) {
   @Override
   public DispatchQueryResponse submit(
       DispatchQueryRequest dispatchQueryRequest, DispatchQueryContext context) {
-    leaseManager.borrow(new LeaseRequest(JobType.BATCH, dispatchQueryRequest.getDatasource()));
```

Comment:
On a side node..we have BatchQueryHandler and Batch Jobtype nomenclature. Even RefreshQueryHandler is also using JobType Batch. We should refactor this nomenclature some time.

Reply:
Removing this would make us loose control over the number of emr jobs created right. I think we should revisit the reason we introduced the lease manager..is it to control number of emr jobs or number of streaming emr jobs?

Reply:
it's for concurrent session / refresh job rule: https://github.com/opensearch-project/sql/blob/3ab785179db593a3999efffb347d897ea8364853/spark/src/main/java/org/opensearch/sql/spark/leasemanager/DefaultLeaseManager.java#L83

it shouldn't be applied to batch job in general. This PR moves this line to RefreshQueryHandler instead